src/applications/personalization/search-representative/config/search/searchRepresentativeWidget.jsx

```diff
@@ -1,49 +1,145 @@
-  }
 
   if (loading) {
     return <div>Loading...</div>;
   } else if (representatives.length > 0) {
     return (
       <div>
-        {representatives.map(option => {
+        <p>
+          Please speak to the service organization or representative before you
+          send your request. You’ll need to mail it to your nearest VA regional
+          office or to the service organization or representative that is going
+          to help you.
+        </p>
+        {currentlyShowingData?.map(option => {
           return (
-            <div
+            <SearchRepresentativeResult
+              option={option}
+              handleClick={handleClick}
               key={option.id}
-              className="vads-u-background-color--gray-lighter vads-u-padding--1p5 vads-u-margin-bottom--1"
-            >
-              <p className="vads-u-font-size--h3 vads-u-font-family--serif vads-u-font-weight--bold">
-                {option.name}
-              </p>
-              <p>{option.city}</p>
-
-              <button
-                onClick={() => handleClick(option.name)}
-                className="usa-button-secondary"
-              >
-                Choose this representative
-              </button>
-            </div>
+            />
           );
         })}
+        <Pagination
+          onPageSelect={page => handleDataPagination(page)}
+          page={currentPage}
+          pages={pages}
+        />
+        <h2>If you don’t see the representative you want</h2>
+        <p>You can go back to make changes and search again.</p>
+        <a
+          className="vads-c-action-link--green"
+          href="/view-change-representative/search/representative-type"
+        >
+          Go back and search again
+        </a>
+        <h2>If you found the representative you want</h2>
+        <p>
+          If you find a representative you want to help you, contact them by
+          phone, email or mail first to be sure they’re available. If you don’t
+          see the representative you want, you can go back to make changes and
+          search again.
+        </p>
+        <h3>Contact the representative you want to help you</h3>
+        <p>
+          When you find a VSO or representative you want to help you, contact
+          them first by phone, mail or email to be sure they’re available.
+        </p>
+        <h3>Let VA know</h3>
+        <p>
+          When you know a VSO or representative is available to help you, there
+          are several ways to make it official.
+        </p>
+        <ul>
+          <li>
+            <p>
+              To have a VSO help you, fill out an Appointment of Veterans
+              Service Organization as Claimant’s Representative (VA Form 21-22).
+            </p>
+            <p>
+              <i
+                className="fas fa-download vads-u-padding-right--1 vads-u-color--link-default"
+                aria-hidden="true"
+              />
+              <a
```

Comment:
Should we be including a `download` attribute / a `type` attribute on these links?

Reply:
I don't think so, they aren't actually for downloading anything, they take you to the find forms page for this form where you can download it.

Reply:
Ok cool, I wasn't sure if this was going to a different page or not